metricbeat/module/memcached/stats/stats.go

```diff
@@ -35,19 +36,36 @@ func init() {
 
 type MetricSet struct {
 	mb.BaseMetricSet
+
+	network    string
+	socketPath string
 }
 
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	config := struct {
+		Network    string `config:"network"`
+		SocketPath string `config:"socket_path"`
```

Comment:
The path should be taken from `hosts`, as we do in other modules where we support unix sockets (like in haproxy or docker).

And the network protocol to use could be obtained from the url too, using tcp by default if not specified.

Reply:
Thanks, fixed.